src/components/NFTViewForm.vue

```diff
@@ -1,113 +1,8 @@
-          <span>Authority</span>
-        </label>
-        <label>
-          <input type="radio" class="nes-radio" value="mint" v-model="chosenMethod" />
-          <span>Mint</span>
-        </label>
-      </div>
-    </div>
-
-    <NotifyInfo v-if="chosenMethod === 'creator' || chosenMethod === 'authority'" class="mt-5">
-      💎 When searching by Creator / Authority you get automatic Rarity Ranking! 💎
-    </NotifyInfo>
-
-    <form @submit.prevent="emitSubmitForm" :class="chosenMethod === 'creator' ? 'mt-5' : 'mt-10'">
-      <div v-if="byAddress" class="nes-field">
-        <div><label for="addr">Wallet Address:</label></div>
-        <input
-          type="text"
-          id="addr"
-          class="nes-input"
-          v-model="owner"
-          :placeholder="DEFAULTS.OWNER"
-        />
-      </div>
-      <div v-else-if="byWallet && !missingWallet" class="nes-field">
-        <div><label for="walletAddr">Your Wallet Address:</label></div>
-        <input
-          type="text"
-          id="walletAddr"
-          class="nes-input text-gray-400"
-          v-model="owner"
-          :placeholder="DEFAULTS.OWNER"
-          :disabled="true"
-        />
-      </div>
-      <div v-else-if="byWallet && missingWallet" class="nes-field">
-        <div><label for="missingWallet">Your Wallet Address:</label></div>
-        <input
-          type="text"
-          id="missingWallet"
-          class="nes-input nes-text is-error"
-          v-model="missingWalletNotice"
-          :disabled="true"
-        />
-      </div>
-      <div v-else-if="byCreator" class="nes-field">
-        <div class="flex">
-          <label for="creator"> Creator Address: </label>
-          <QuestionMark @click="showModal('tooltipCreator')" />
-        </div>
-        <input
-          type="text"
-          id="creator"
-          class="nes-input"
-          v-model="creator"
-          :placeholder="DEFAULTS.CREATOR"
-        />
-      </div>
-      <div v-else-if="byAuthority" class="nes-field">
-        <div><label for="authority">Update Authority Address:</label></div>
-        <input
-          type="text"
-          id="authority"
-          class="nes-input"
-          v-model="authority"
-          :placeholder="DEFAULTS.AUTHORITY"
-        />
-      </div>
-      <div v-else-if="byMint" class="nes-field">
-        <div><label for="mint">Mint Address:</label></div>
-        <input
-          type="text"
-          id="mint"
-          class="nes-input"
-          v-model="mint"
-          :placeholder="DEFAULTS.MINT"
-        />
-      </div>
-
-      <div class="flex justify-between mt-5">
-        <button
-          class="nes-btn is-primary"
-          :class="{ 'is-disabled': isLoading }"
-          :disabled="isLoading"
-          type="submit"
-        >
-          Load NFTs
-        </button>
-        <slot />
-      </div>
-    </form>
+    <div></div>
+    <div></div>
+    <p class="title"> Open Questions</p>
```

Comment:
i would remove this from this file and move it to NFTViewView

Reply:
that way any one else who uses this form wont see it

Reply:
will do!